reswift-jobs/reswift-jobs/UI/JobViewController.swift

```diff
@@ -60,18 +60,13 @@ class JobViewController: NSViewController {
        }
    }
 
-   @IBAction func changeTitle(_ sender: AnyObject) {
+    @IBAction func changeTitle(_ sender: AnyObject) {
 
```

Comment:
extraneous space before `@IBAction`

Reply:
Heh. It's actually at the right tab position. The rest of the file needs a reformatting, which I want to avoid for the time being.